plugins/source-kafka/pkg/factories/command_factory.go

```diff
@@ -55,22 +55,23 @@ func (f *kafkaSourceCommandFactory) SourceCommand() *cobra.Command {
 	sourceCmd := f.defaultCommandFactory.SourceCommand()
 	sourceCmd.Use = "kafka"
 	sourceCmd.Short = "Knative eventing Kafka source plugin"
-	sourceCmd.Long = "Manage your Knative Kafka eventing sources"
+	sourceCmd.Long = "Manage Knative Kafka eventing sources"
 	return sourceCmd
 }
 
 func (f *kafkaSourceCommandFactory) CreateCommand() *cobra.Command {
 	createCmd := f.defaultCommandFactory.CreateCommand()
-	createCmd.Short = "create NAME"
+	createCmd.Use = "create NAME --sink SINK"
+	createCmd.Short = "Create a kafka source"
 	createCmd.Example = `#Creates a new Kafka source named as 'mykafkasrc' which subscribes a Kafka server 'my-cluster-kafka-bootstrap.kafka.svc:9092' at topic 'test-topic' using the consumer group ID 'test-consumer-group' and sends the event messages to service 'event-display'
```

Comment:
Should we add a space between ```#``` and example description? I just found ```kn``` command has space between them.

Reply:
Fixed. Thank you.